src/reachy2_sdk/parts/arm.py

```diff
@@ -287,6 +287,52 @@ def inverse_kinematics(
             answer = np.round(answer, round).tolist()
         return answer
 
+    def get_default_pose_matrix(self, common_pose: str = "default") -> npt.NDArray[np.float64]:
+        """Return the 4x4 pose matrix of default robot poses."""
+        if common_pose not in ["default", "elbow_90"]:
+            raise ValueError(f"common_pose {common_pose} not supported! Should be 'default' or 'elbow_90'")
+
+        if common_pose == "elbow_90":
```

Comment:
This is also defined in set_pose. We should factorize the poses somewhere otherwise we may do a mistake by changing one and not the other.